### ISSUE TITLE
When SSHing into a KubevirtCI, long terminal commands are not wrapped, therefore looking weird

### ISSUE THREAD

**Issue by iholder101** [kubevirt/kubevirt#12832]
**What happened**:
When SSHing into a KubevirtCI node, if the command is long enough, it is not warpped by instead being messy overriding the beginning of the command:

```shell
> ./cluster-up/ssh.sh node02
Last login: Sun Sep 15 08:35:14 2024 from 192.168.66.2

# This is short enough, looking good
[vagrant@node02 ~]$ find -regex '.*pod.*' -type d -exec cat {}/memory.max \;
[vagrant@node02 ~]$

# Adding `&& ls {}` at the end of the find exec command
{} \;t@node02 ~]$                                                            s \;                                                                          l 
\;agrant@node02 ~]$ find -regex '.*pod.*' -type d -exec cat {}/memory.max &&  \;
```

This is how it looks like:
![image](https://github.com/user-attachments/assets/98d6d07f-0c44-423e-b25d-08e556fed587)

**What you expected to happen**:
The command to be wrapped into a new line in a human-readable way.

**How to reproduce it (as minimally and precisely as possible)**:
As explained above. 100% reproducible.

**Environment**:
- KubeVirt version (use `virtctl version`): v1.3 (main)

**Comment by Acedus** [comment]
Seems like a PTY issue along with `SIGWINCH` not being handled properly. 
Execing into the Pod and then running `ssh.sh` seems to work fine.
Also verified this by doing the following:
```
$ podman exec -it <kubevirtci container> sh
$ trap 'echo test' WINCH
```

Resizing the terminal doesn't do anything.

**Comment by akalenyu** [comment]
TBH this may be a general RFE for virtctl console (virtctl ssh seems to be aware according to dosu)

Back to the subject, for the kubevirtci ssh script, I don't think we're aware of SIGWINCH or try to do any resizing at all.
As a quick W/A one could run `stty size` on the host machine and apply the same values on the kvci node

**Comment by brianmcarey** [comment]
@iholder101  - does this belong more to kubevirt/kubevirtci? I can transfer the issue if you want?

**Comment by iholder101** [comment]
> @iholder101 - does this belong more to kubevirt/kubevirtci? I can transfer the issue if you want?

Yes, feel free to do so if you think that's more appropriate! Thanks!
